### DISCUSSION TITLE
How can I manually erase or make a partition?

### DISCUSSION THREAD

**Discussion by 47louis** [elementary/installer#736] [Q&A]
### What Happened?

I had Ubuntu 23.04 I try to install elemantory 7.0 no luck I get a gray screen. then I download 7.1 I can try 7.1 and push button to install computer say I can't install because I can't erase or make a partition. Is their in 7.1 a possibility to do this manualy?
Erase disk or format the disk?

### Steps to Reproduce

Try to install 7.0 I bought 7.0 early this year. Look of I like it.
Install procedure very long time , then a gray screen.
Then I try to install 7.1 with the results see above.

### Expected Behavior

I want to work with 7.1

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Hardware Info

notebook clevo 32 Gb, Intel i7 2.80 GHz. I use the graphic from the cpu. Videocard death. No issyes with other Linux distro.

**Comment by danirabbit** [comment]
You can manually manage partitions as part of the custom install step.

On this page, select "Custom Install"

![Screenshot from 2023-11-06 10 49 48](https://github.com/elementary/installer/assets/7277719/8d3104bb-f3ef-410f-abcb-63c9d8f2344b)

Then select "Modify Partitions" in the bottom corner

![Screenshot from 2023-11-06 10 49 59](https://github.com/elementary/installer/assets/7277719/fe24e54f-4ae7-476f-b119-fb3106b44761)

After modifying partitions and applying changes, you can either proceed with a custom install or go back and try a standard install when the disk has been erased

**Reply by 47louis** [reply]
Thanks fot answer, I gone to try it.

**Reply by mimaguire** [reply]
Can "Modify Partitions" remove the MacOS X Base System partition?  I can't get the intel Mac with i5 processor to launch elementary 8.0

**Reply by ryonakano** [reply]
> Can "Modify Partitions" remove the MacOS X Base System partition?

It just launches GParted and I've successfully clean installed elementary OS 8 on my MacBook Air 2010 for many times so I think yes.

> I can't get the intel Mac with i5 processor to launch elementary 8.0

Is your Mac able to boot elementary OS 8 from an installation media but isn't possible to boot from SSD, right? If so I think you can check if the OS X Base System partition disappears or still alive by booting from the installation media again and launching GParted.
